src/dsymbol.d

```diff
@@ -172,6 +172,8 @@ enum : int
     IgnorePrivateMembers    = 0x01, // don't find private members
     IgnoreErrors            = 0x02, // don't give error messages
     IgnoreAmbiguous         = 0x04, // return NULL if ambiguous
+    IgnoreImports           = 0x08, // don't find symbols in imports
+    IgnoreThisModule        = 0x10, // only look in imports
```

Comment:
> dsymbol.h  also needs to be updated.

There is no reason for the C++ code to ever need these values. Putting it in dsymbol.h would imply a reason.